vertx-web-graphql/src/test/java/io/vertx/ext/web/handler/graphql/JsonResultsTest.java

```diff
@@ -60,21 +60,6 @@ public DataFetcher getDefaultDataFetcher(FieldWiringEnvironment environment) {
       .build();
   }
 
-  @Override
-  protected Object getAllLinks(DataFetchingEnvironment env) {
-    @SuppressWarnings("unchecked")
-    List<Link> links = (List<Link>) super.getAllLinks(env);
-    return links.stream()
-      .map(link -> {
-        JsonObject jsonObject = new JsonObject()
-          .put("url", link.getUrl())
-          .put("description", link.getDescription())
-          .put("userId", link.getUserId());
-        return jsonObject;
-      })
-      .collect(JsonArray::new, JsonArray::add, JsonArray::addAll);
-  }
-
```

Comment:
These tests are also failing in the `master` branch and this method seems to be the culprit

Reply:
Can you elaborate? The `master` branch CI build passes.